Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -405,8 +405,7 @@ public override void StartDriver()
                     case eBrowserType.Chrome:
                         ChromeOptions options = new ChromeOptions();
 
-                        if (BrowserMinimized == false)
-                            options.AddArgument("--start-maximized");
+                        options.AddArgument("--start-maximized");
```

Comment:
why BrowserMinimized was removed here?

Reply:
It's removed because many of the Web applications have different css properties for different views and maximized browser exposes each & every element that exist on the page. On the other hand, '_BrowserMinized_' property just minimizes the browser once it is launched, thus, both the actions occurs one after the other as,adding argument "--start-maximized" launches the browser in maximized window, once finished, browser will be minimized if '_BrowserMinized_' is set to '_true_'.